src/EnergyPlus/CMakeLists.txt

```diff
@@ -700,8 +699,20 @@ set(INPUTPARSING_SRC
 
 create_src_groups("${INPUTPARSING_SRC}")
 
-add_custom_target(
-  GenerateEmbeddedEpJSONSchema
+configure_file("${PROJECT_SOURCE_DIR}/idd/Energy+.idd.in" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.idd")
+
+add_custom_command(
+        OUTPUT "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Energy+.schema.epJSON"
+        COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/scripts/dev/generate_epJSON_schema/generate_epJSON_schema.py" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
```

Comment:
This line is going to call generate_schema with `"${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"` as the only argv.  This is fine because the current IDD will have been copied in there by this time.  So then generated schema should find the IDD in there and create an EpJSON schema in there as well.